### ISSUE TITLE
22Mars - Simplify edge connections

### ISSUE THREAD

**Issue by scottredracecar** [tobymao/18xx#11131]
Alex, I find it hard to follow the map edge connections on the current map. Certainly for coding this, it's a good way to do it, but maybe not so good for playing it.
https://18xx.games/map/22mars

Maybe you have something planned for this for the future. Perhaps some colored off-board spikes would be a better way to display this?

<img width="776" alt="image" src="https://github.com/user-attachments/assets/f34df7f9-5df1-49e6-987c-d387abbbc207">

**Comment by philcampeau** [comment]
It also looks like there's a missing connection in B3 for the middle track. 

But yes agreed that a color code might be best. Or maybe a letter/color system?

**Comment by vandamm** [comment]
Originally Jonas has letter+color on the outside frames, but I found it not that understandable. I'll experiment with making the connections more pale and color-coded
